subprojects/libero-distribution/README.md

```diff
@@ -37,6 +37,72 @@ installed distribution will be under `build/install`
   * config files can be specified as a parameter
   * searches in `<projectdir>/.libero.yml`
   * searches in `~/.libero/<project-name>/.libero.yml`
+  
+## Cleaning local artifacts
+
+Libero now supports cleaning artifacts from the local repo prior to building the project
+
+Artifacts can be cleaned by providing the `project.maven.repo.clean` property with a list of comma separated group ids
+Any artifact located at or below that group id depth will be deleted
+
+## Using an alternate local maven repository
+
+An alternate local maven repo can be configured by providing the `project.maven.repo.local` property with a path to use for a local repository
```

Comment:
What is everyone's thoughts on generating a new local repo directory for each release by default?  Then there would be an override to use the user's .m2 repo or a given directory.  I think that would be a safer default behavior.